ttg/ttg/util/dot.h

```diff
@@ -10,6 +10,8 @@
 namespace ttg {
   /// Prints the graph to a std::string in the format understood by GraphViz's dot program
   class Dot : private detail::Traverse {
+    bool disable_type=true;
```

Comment:
this needs to be configurable by user by giving parameter to `Dot`'s constructor (with default behavior, i.e. when `disable_type ` is not given explicitly, being the same behavior as before, i.e. types being enabled).